.gitignore

```diff
@@ -0,0 +1 @@
+node_modules/
```

Comment:
- [ ] As the requirements suggest, consider adding your `test.md` file with the text `"This file should be ignored by git"` inside your `.gitignore`. Make sure that this file is actually ignored by git and is not present in your GitHub repository